tests/test_flow_helpers.py

```diff
@@ -0,0 +1,95 @@
+
+import pytest
+import os
+import re
+import test_helper
+
+# Import load_testing files
+# :/ this kind of import only works when you run `pytest` from the root of the project.
+import sys
+sys.path.append('./load_testing')
+from common_flows.flow_helper import *
```

Comment:
I don't do a lot of Python, so would love to hear other opinions on this, but I think it would be clearer to indicate *what* we're testing if we didn't do a `*` import? Like we explicitly referenced `flow_helper.authenticity_token()` or something? Or maybe the explicit frmo `common_flows.flow_helper import authenticity_token, ...` form?

Reply:
Good point.  I've been using `*` while developing cause it's easier.  My editor doesn't like it either, I'll fix :)